feat: added CLI scripts documentation (#4223)
influxdata/docs-v2: content/influxdb/v2.3/reference/cli/influx/scripts/invoke.md

---
title: influx scripts invoke

> description: The `influx scripts invoke` command runs a script in InfluxDB.

Reviewer: ```suggestion
description: The `influx scripts invoke` command executes an invokable script in InfluxDB.
```

menu:
  influxdb_2_3_ref:
    name: influx scripts invoke
    parent: influx scripts
weight: 201
---

> The `influx scripts invoke` command runs a script in InfluxDB.

Reviewer: ```suggestion
The `influx scripts invoke` command executes an invokable script in InfluxDB.
```

| `-c` | `--active-config` | CLI configuration to use for command                                  | string     |                       |
|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`) | string     | `INFLUX_CONFIGS_PATH` |
| `-f` | `--file`          | File name containing the script parameters, in JSON                   | string     |                       |
| `-h` | `--help`          | Help for the `delete` command                                         |            |                       |
|      | `--hide-headers`  | Hide table headers (default `false`)                                  |            | `INFLUX_HIDE_HEADERS` |
|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            | string     | `INFLUX_HOST`         |
|      | `--http-debug`    | Inspect communication with InfluxDB servers.                          | string     |                       |
|      | `--json`          | Output data as JSON (default `false`)                                 |            | `INFLUX_OUTPUT_JSON`  |

> | `-p` | `--params`        | JSON string containing the parameters                                 | string     |                       |

Reviewer: ```suggestion
| `-p` | `--params`        | JSON string containing script parameters                              | string     |                       |
```
